README.md

```diff
@@ -1,43 +1,45 @@
 <h1 align="center">5GMS Application Provider</h1>
 <p align="center">
   <img src="https://img.shields.io/badge/Status-Under_Development-yellow" alt="Under Development">
-  <img src="https://img.shields.io/github/v/tag/5G-MAG/rt-5gms-application-function?label=version" alt="Version">
```

Comment:
I actually wanted to show the Application Function version here, since the management UI is dependent from it. Maybe adding both...?

Reply:
Management UI supports all M1 operations for Application Function **v1.4.1** or later. However, seems more reasonable to add this information into the Readme, and to replace AF badge with AP version.
So this change is good.

Reply:
@stojkovicv Oki! If you want keep updating the documentation branch, I won't work on it for the time being. Once we get confirmation from Daniel, I will sync my documentation branch, make further changes and resend the PR if needed.